Buildings/Controls/OBC/ASHRAE/G36/AHUs/SingleZone/SetPoints/ReliefDamper.mo

```diff
@@ -0,0 +1,208 @@
+  connect(relDamPos.y, swi1.u1)
+    annotation (Line(points={{41,0},{48,0},{48,42},{58,42}},
+      color={0,0,127}));
+  connect(uOutDamPos, greThr.u)
+    annotation (Line(points={{-120,0},{-80,0},{-80,20},{-62,20}},
+      color={0,0,127}));
+  connect(minPosAtMinSpd.y, relDamPos.x1)
+    annotation (Line(points={{-19,-20},{-12,-20},{-12,8},{18,8}},
+      color={0,0,127}));
+  connect(minRelDam.y, relDamPos.f1)
+    annotation (Line(points={{-59,-40},{-4,-40},{-4,4},{18,4}},
+      color={0,0,127}));
+  connect(swi1.y, yRelDamPos)
+    annotation (Line(points={{81,50},{100,50},{100,0},{130,0}},
+      color={0,0,127}));
+
+annotation (
+  defaultComponentName = "reliefDamper_singleZone",
+  Icon(graphics={Rectangle(
+        extent={{-100,-100},{100,100}},
+        lineColor={0,0,127},
+        fillColor={255,255,255},
+        fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-96,78},{-56,42}},
+          lineColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uBuiPre"),
+        Text(
+          extent={{-96,-42},{-52,-78}},
+          lineColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uSupFan"),
+        Text(
+          extent={{34,22},{96,-18}},
+          lineColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="yRelDamPos"),
+        Polygon(
+          points={{-80,92},{-88,70},{-72,70},{-80,92}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(points={{-80,80},{-80,-88}}, color={192,192,192}),
+        Line(points={{-90,-78},{82,-78}}, color={192,192,192}),
+        Polygon(
+          points={{90,-78},{68,-70},{68,-86},{90,-78}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(points={{-80,-78},{-80,-78},{14,62},{80,62}}, color={0,0,127}),
+        Text(
+          extent={{-100,140},{100,100}},
+          lineColor={0,0,255},
+          textString="%name")}),
+   Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{120,
+            100}})),
+ Documentation(info="<html>      
+<p>
+This sequence controls actuated relief dampers (<code>yRelDamPos</code>) 
+without fans. It is implemented according to ASHRAE Guidline 35 (G36), PART5.N.8. 
+(for multiple zone VAV AHU), PART5.P.6 and PART3.2B.3 (for single zone VAV AHU).
+</p>   
+
+<h4>Single zone VAV AHU: Control of actuated relief dampers without fans(PART5.P.6)</h4>
+<ol>
+<li>Relief damper position setpoints (PART3.2B.3)
+<ul>
+<li><code>minRelDamPos</code>: The relief damper position that maintains a building 
+pressure of 12 Pa (0.05”) while the system is at MinPosMin (i.e., the economizer 
+damper is positioned to provide MinOA while the supply fan is at minimum speed).</li>
```

Comment:
write out the abbreviations, or use `<code>...</code>` if you refer to a variable name